### PR TITLE
Fix Remove allocation Issue

### PR DESCRIPTION
**Pull request by udaydenduluri33**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-4929
https://tools.hmcts.net/jira/browse/EUI-4931
https://tools.hmcts.net/jira/browse/EUI-4754
https://tools.hmcts.net/jira/browse/EUI-4935
https://tools.hmcts.net/jira/browse/EUI-4938

### Change description ###

1. Fix for the Remove allocation URL from My Work, All Work
2. Handling errors with Exclusions with  Judicial users was fixed
3. Judicial Users domain was passed incorrectly was fixed
4. Judicial users name was not getting displayed on Exclusions was fixed
5. Remove Exclusion For Judicial users missing user name was fixed
6. After successful Remove Allocation [Exclusion] taking the user the previous page is fixed

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

File: api/workAllocation2/util.ts [hunk 1]
```diff
@@ -456,13 +456,15 @@ export function mapRoleCaseData(roleAssignment: RoleAssignment, caseDetail: Case
...
     case_category: caseDetail.case_data.appealType,
+    case_type: caseDetail.case_type_id,
     case_id: caseDetail.id,
     case_name: caseDetail.case_data && caseDetail.case_data.caseName ? caseDetail.case_data.caseName : caseDetail.id,
     case_role: roleAssignment.roleName,
     role: roleAssignment.roleName,
     endDate: roleAssignment.endTime,
     id: roleAssignment.id,
     jurisdiction: caseDetail.jurisdiction,
+    jurisdictionId: caseDetail.jurisdiction,
```
Comment (kuda1992) [inline api/workAllocation2/util.ts:467]: Why do we need `jurisdictionId` if it's just going to be assigned to `jurisdiction`

**Review by johnbenjamin-hmcts — APPROVED** [review]
Approved

**Review by kuda1992 — CHANGES_REQUESTED** [review]
Left some comments confused why we need jurisdiction Id

File: api/workAllocation2/util.ts (hunk 1 above)
Comment (udaydenduluri33) [reply api/workAllocation2/util.ts:467]: jurisdictionId - IA
jurisdiction - Immigration And Asylum
We need both @kuda1992

Comment (kuda1992) [reply api/workAllocation2/util.ts:467]: Understood the pr is failing unit tests thou.

**Review by kuda1992 — APPROVED** [review]
Thanks for clarifying @udaydenduluri33

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Pull request merged**